src/main/java/com/alkemy/ong/entity/SlideEntity.java

```diff
@@ -27,5 +27,5 @@
     private Integer order;
     @ManyToOne (fetch = FetchType.LAZY)
     @JoinColumn(name = "organization_id")
-    private String organizationId;
+    private OrganizationEntity organizationId;
```

Comment:
Por que se le da el nomobre de organizationId a un objeto OrganizationEntity, en todo caso el atributo deberia ser organization. Favor revisar nomenclatura de nombres

Reply:
Name reviewed and change for OrganizationEntity, imported class also correct softDelete query and clause's name